OPERATIONS.md

```diff
@@ -4,34 +4,28 @@
 
 ### Deploying Every Service
 
-This series of commands will deploy non-prod every service for you. Run them from the top level directory (where this file is located)
+This series of commands will deploy non-prod every service for you. Run them from the top level directory (where this file is located). If you want to run them all quickly, then run each block of bash in a new terminal. If you want to be more careful, run them all one at a time, from top to bottom, inspecting the output on every step.
```

Comment:
I'm inclined the "fast" description because if you need to run every test "slow" then its going to take you 30 minutes to even check on very small iterations. And that's not always viable.